applications/views.py

```diff
@@ -44,7 +45,9 @@ def get_queryset(self):
             return BootcampApplication.objects.prefetch_state_data()
         else:
             return (
-                BootcampApplication.objects.filter(user=self.request.user)
+                BootcampApplication.objects.prefetch_state_data()
+                .filter(user=self.request.user)
+                .annotate(num_orders=Count("orders", status=Order.FULFILLED))
```

Comment:
Are you using this annotation anywhere?

Reply:
No, it was a different attempt at counting fulfilled orders but I couldn't figure how to get this to the serializer.